README.md

```diff
@@ -117,8 +123,8 @@ The `setup.py clean` call mentioned above will also clean out the native extensi
 #### Running Unit Tests
 
 1. Open a shell in the PyPRT git root.
-1. First time only: run `pipenv install` to get all required Python packages.
-1. Run `pipenv run tox`.
+1. First time only: setup a virtual Python environment with build dependencies for PyPRT, see "Build Python Wheel" above.
```

Comment:
Maybe replace the "above" by "[above] (#build-python-wheel)" (without the white space).

Reply:
done